cmd/kyma/alpha/deploy/opts.go

```diff
@@ -165,3 +172,11 @@ func (o *Options) pathExists(path string, description string) error {
 func (o *Options) workspaceTmpDir() string {
 	return filepath.Join(o.WorkspacePath, "tmp")
 }
+
+func getDefaultWorkspacePath() string {
+	kymaHome, err := files.KymaHome()
+	if err != nil {
+		return filepath.Join(".kyma-sources")
```

Comment:
Is there a reason to use `filepath.Join()` here?
When the parameter list is < 2 it does nothing.

Reply:
yes, it is not needed here. I removed it.